src/graph/graph.c

```
@@ -1408,6 +1408,105 @@
+	Delta_Matrix D;
+	GrB_Info     info;
+
+	//--------------------------------------------------------------------------
+	// Graph's adjacency matrix
+	//--------------------------------------------------------------------------
+
+	D = Graph_GetAdjacencyMatrix(g, false);
+	M = Delta_Matrix_M(D);
+
+	info = GxB_Matrix_memoryUsage(&n, M);
+	ASSERT(info == GrB_SUCCESS);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)